src/test/java/com/commercetools/project/sync/model/LastSyncCustomObjectTest.java

```diff
@@ -0,0 +1,142 @@
+package com.commercetools.project.sync.model;
```

Comment:
could you assert .hashCode() methods after asserting equals ?
sth like this: ```assertThat(lastSyncCustomObject.hashCode()).isEqualTo(otherLastSyncCustomObject.hashCode());```

Reply:
but why? shouldn't the .equals be covering that?

Reply:
Your `equals` implementation is not using `hashCode`. If one deletes your `hashCode` implementation the class won't work as expected in hash-based collections

Reply:
for to cover it better 😉

Reply:
true. You're right. Thanks. 
600f0c6